docs/quickstart/data_conventions.md

```diff
@@ -0,0 +1,76 @@
+# Data conventions
+
+## Coordinate conventions
+
+Here we explain the coordinate conventions for using our repo.
+
+### Camera/view space
+
+We use the OpenGL/Blender (and original NeRF) coordinate convention for cameras. +X is right, +Y is up, and +Z is pointing back and away from the camera. -Z is the look-at direction. Other codebases may use the COLMAP/OpenCV convention, where the Y and Z axes are flipped from ours but the +X axis remains the same.
+
+### World space
+
+Our world space is oriented such that the up vector is +Z. The XY plane is parallel to the ground plane. In the viewer, you'll notice that red, green, and blue vectors correspond to X, Y, and Z respectively.
+
+<hr>
+
+## Dataset format
+
+Our explanation here is for the nerfstudio data format. The `transforms.json` has a similar format to [Instant NGP](https://github.com/NVlabs/instant-ngp).
+
+**Camera intrinsics**
+
+At the top of the file, we specify the camera intrinsics. We assume that all the intrinsics parameters are the same for every camera in the dataset. The following is an example
+
+```json
+{
+  "fl_x": 1072.281897246229, // focal length x
+  "fl_y": 1068.6906965388932, // focal length y
+  "cx": 1504.0, // principal point x
+  "cy": 1000.0, // principal point y
+  "w": 3008, // image width
+  "h": 2000, // image height
+  "camera_model": "OPENCV_FISHEYE", // camera model type
+  "k1": 0.03126218448029553, // first radial distorial parameter
+  "k2": 0.005177020067511987, // second radial distorial parameter
+  "k3": 0.0006640977794272005, // third radial distorial parameter
+  "k4": 0.00010067035656515042 // fourth radial distorial parameter
```

Comment:
There is also "p1" and "p2" tangential distortions.
Fisheye uses k1-4, perspective uses k1-2,p1-2